src/neuroconv/tools/roiextractors/roiextractors.py

```diff
@@ -797,7 +804,7 @@ def add_fluorescence_traces(
 
         # Build the roi response series
         roi_response_series_kwargs.update(
-            data=np.array(trace).T,
+            data=H5DataIO(SliceableDataChunkIterator(trace, **iterator_options), **compression_options),
```

Comment:
This is temporary - an actual `SegmentationExtractorDataChunkIterator(GenericDataChunkIterator)` would be preferred, but best in follow-up PR.